parserator_web/templates/parserator_web/index.html

```diff
@@ -17,7 +17,12 @@ <h3 id="usaddress-parser"><i class="fa fa-fw fa-map-marker-alt"></i> U.S. addres
           <button id="submit" type="submit" class="btn btn-success mt-3">Parse!</button>
         </form>
       </div>
-      <!-- TODO: Display parsed address components here. -->
+      <!-- User friendly error message, with more technical details collapsed -->
+      <div id="error-message" style="display:none">
+        <p>Uh oh! That doesn't seem to be a valid address.</p>
+        <button data-toggle="collapse" data-target="#error-info">Find out more</button>
```

Comment:
How about adding [Bootstrap classes](https://getbootstrap.com/docs/4.0/components/buttons/) to this button so it fits with the aesthetic of the rest of the site?